game/i18n.hh

```diff
@@ -1,42 +1,27 @@
 #pragma once
 
-#ifdef USE_GETTEXT
-/* Internationalization Dependances */
-    #ifdef __APPLE__ // Using system gettext can choke portability of an app-bundle. 
-        #include "libintl.h"
-        #include "locale.h"
-    #else
-        #include <libintl.h>
-        #include <locale.h>
-    #endif
+#include <boost/locale.hpp>
+#include <iostream>
 #include "fs.hh"
-#define _(x) gettext(x)
-#else
-#define _(x) (x)
-#endif
+using namespace std;
+using namespace boost::locale;
+
+#define _(x) translate(x).str()
+#define gettext_noop(x) x
 
 class Gettext {
```

Comment:
Qt-creator is quite good, just load your cmakelist.txt in there and it will take care of everything else! you can even do custom launch scripts where you can add "gksu make install" to automate that part and have the debugger work perfectly.